satpy/tests/utils.py

```diff
@@ -355,3 +355,100 @@ def __call__(self, dsk, keys, **kwargs):
             raise RuntimeError("Too many dask computations were scheduled: "
                                "{}".format(self.total_computes))
         return dask.get(dsk, keys, **kwargs)
+
+
+def make_a_scene(cont_dict, daskify=False, area=True, common_attrs=None):
```

Comment:
The name of the function is fun and humorous, however I think it would be ease maintainability if it had a clearer name, like `fake_scene` for instance.

Reply:
Renamed to `make_a_fake_scene`.